app/api/datastore/entities/v1/maat/income_details.rb

```diff
@@ -13,8 +13,33 @@ class IncomeDetails < Grape::Entity
           expose :manage_without_income, expose_nil: false
           expose :manage_other_details, expose_nil: false
 
+          OTHER_INCOME_PAYMENTS = %w[
+            student_loan_grant
+            board_from_family
+            rent
+            financial_support_with_access
+          ].freeze
+
+          OTHER_INCOME_BENEFITS = %w[
+            jsa
+          ].freeze
```

Comment:
Might be useful to add a comment here explaining why we have these are combined

Reply:
Also not sure i follow why other is not included in these arrays

Reply:
Also think `OTHER_INCOME_PAYMENTS`,  `OTHER_INCOME_BENEFITS` and `OtherPaymentCalculator` should be reworded as we have an other payment and benefit type so it gets confusing. Although I understand this mirrors the wording on maat

Reply:
@hiboabd I have refactored and followed SRP.